src/compute-client/src/command.proto

```diff
@@ -51,6 +51,13 @@ message ProtoComputeCommand {
 message ProtoInstanceConfig {
     mz_compute_client.logging.ProtoLoggingConfig logging = 1;
     uint64 replica_id = 2;
+    optional ProtoCommunicationConfig communication_config = 3;
```

Comment:
Protobuf encoding has now changed again, since `CreateTimely` and `CreateInstance` are now separate commands, `ProtoInstanceConfig` remains unchanged.